plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx

```diff
@@ -19,9 +19,12 @@
 
 import React from 'react';
 import { t } from '@superset-ui/translation';
-import { controls } from '@superset-ui/chart';
-
-const { formatSelectOptions, D3_TIME_FORMAT_OPTIONS, D3_FORMAT_DOCS, D3_FORMAT_OPTIONS } = controls;
+import {
+  formatSelectOptions,
+  D3_TIME_FORMAT_OPTIONS,
+  D3_FORMAT_DOCS,
+  D3_FORMAT_OPTIONS,
+} from '@superset-ui/chart';
```

Comment:
Please see #417. May have to change reference after that one is merged.

Reply:
Let's merge that and I'll rebase